config/initializers/smtp.rb

```diff
@@ -0,0 +1,13 @@
+#if Rails.env.development?
+ActionMailer::Base.delivery_method = :smtp
+ActionMailer::Base.smtp_settings = {
+  :address              => "smtp.zoho.com",
+  :port                 => 465,
+  :user_name            => 'countries_launch_academy@zoho.com',
```

Comment:
Use the new Ruby 1.9 hash syntax.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.